src/TreeList/TreeList.tsx

```diff
@@ -9,6 +9,7 @@ import { ListFocus } from '../ListFocus'
 import { StyledList } from '../SelectList'
 import { BlockProps } from '../Block'
 import styled from 'styled-components'
+import AutoSizer from 'react-virtualized-auto-sizer'
```

Comment:
Do we need to explicitly add this to our dependencies?

Reply:
Yeah, it's a separate package.

Reply:
Got it.